src/app/components/splash/splash.component.html

```diff
@@ -1,12 +1,31 @@
 <div class="splashpage">
-  <div class="whorwe">
+  <div class="whorwe" [style.backgroundSize]="'cover'">
       <h1>Who is the Voters' Voice Initiatives?</h1>
-      <p>The Voters’ Voice Initiatives are aimed at providing “we the people” a greater voice in our government. Transparency in campaign finance is critical.  Providing full disclosure on the flow of money in local politics empowers citizens to hold their government accountable for decisions made. Transparency reduces the potential for corruption and increases trust between voters and their government.</p>
+      <p class="vv-description">The Voters’ Voice Initiatives is a non-partisan coalition (including <a href="https://www.lwv.org/">Represent.us</a> and the 
```

Comment:
Looks like these links got swapped.

Reply:
Whooops sorry @nancyheiss. Should be fixed now.